File: Morning Snack/Python/amstrong_number_range.py
for count in range(1, 1000):
    new_number  = count
    lengthOfNumbers = str(new_number)
    length = len(lengthOfNumbers)
    exponential = 1
    sum_of_number = 0
    digit = 1
    while(new_number != 0):
        digit = new_number % 10;
        exponential = digit**3
        sum_of_number += exponential
        new_number = new_number/10  

if(sum_of_number == count):
    System.out.println(count)

File: Morning Snack/Java/AmstrongNumberRange.java
public class AmstrongNumberRange{
    public static void main(String[]args){
    
        for(int count=1; count <= 1000; count++){
        int newNumber  = count;
        String lengthOfNumbers = String.valueOf(newNumber);
        int length = lengthOfNumbers.length();
        double exponential;
        double sum = 0;
        double digit ;
        while(newNumber != 0){
               digit = newNumber % 10;
               exponential = Math.pow(digit, length);
               sum += exponential;
               newNumber = newNumber/10;  
        }
        if(sum == count){
                 System.out.println(count );
    }
    }
      
    
    }
}
